data/categories.txt

```diff
@@ -1026,6 +1026,7 @@ fa:بقالی
 
 shop-deli|@food
```

Comment:
@vng I meant that [type.shop.deli] in types_strings.txt for example has all translations. Here are only a few of them.

Reply:
Fixed